src/kani-compiler/rustc_codegen_kani/src/utils/names.rs

```diff
@@ -109,6 +109,10 @@ impl<'tcx> GotocCtx<'tcx> {
         // TODO: use def_id https://github.com/model-checking/kani/issues/364
         idx.to_string().into()
     }
+
+    pub fn reach_check_msg(check_id: &str) -> String {
```

Comment:
What does the `check_id` represent?

Reply:
This is a unique identifier for each check generated by codegen. I added documentation for the function that should hopefully clarify.

Reply:
Have you considered using the metadata file instead to do the mapping?

Reply:
@danielsn and I discussed this option yesterday. The difficulty here is that the output from CBMC wouldn't include this metadata, so the CBMC parser script will need to parse the metadata file and use it when parsing the CBMC output.